src/script/descriptor.cpp

```diff
@@ -322,6 +322,8 @@ class ConvertorDescriptor : public Descriptor
 };
 
 CScript ConvertP2SH(const CScript& script) { return GetScriptForDestination(CScriptID(script)); }
+ColorIdentifier colorID = ColorIdentifier();
+CScript ConvertCP2SH(const CScript& script) { return GetScriptForDestination(CScriptID(script), &colorID); }
```

Comment:
`ConvertCP2SH` does not appear to be used.

Reply:
i will remove this code since it is not being used